src/components/Button/Button.js

```diff
@@ -19,19 +23,19 @@ Primary buttons should be used for the principle call to action on the page. Mod
 Small buttons may be used when there is not enough space for a regular sized button. This issue is most found in tables. Small button should have three words or less.
 `;
 
-Button.defaultProps = {
-	children: 'Click me!',
-};
-
+//This allows for the definition of rules that each prop type has to follow in order to be used properly
 Button.propTypes = {
-	/** The text shown inside the button. */
 	children: PropTypes.string.isRequired,
-
-	/** If the button should be disabled and unclickable. */
+	variant: PropTypes.oneOf(['primary', 'secondary']).isRequired,
+	onClick: PropTypes.func.isRequired,
 	disabled: PropTypes.bool,
+};
```

Comment:
Can you add comments for each of the proptypes in ``/** Description of proptype */`` format? (see [this for example](https://github.com/Flujible/ethereum-pos/blob/133a82d1739102ceb07afc746c18bddb2b919ef6/src/components/AddressInput/AddressInput.js))